src/HCS.Meta.Robots/Composers/AddRobotsHeaderComposer.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Umbraco.Cms.Core.Composing;
+using Umbraco.Cms.Core.DependencyInjection;
+using Umbraco.Cms.Web.Common.ApplicationBuilder;
+
+namespace HCS.Meta.Robots.Composers
+{
+    internal class AddRobotsHeaderComposer : IComposer
+    {
+        public void Compose(IUmbracoBuilder builder)
+        {
+            builder.Services.Configure<UmbracoPipelineOptions>(options => {
```

Comment:
Can you move this logic into the UmbracoBuilderExtensions please, this is so everything is registered together.
https://github.com/NikRimington/HCS.Meta.Robots/blob/8f88018208c0cfe8a40e142323723dee938d70d9/src/HCS.Meta.Robots/UmbracoBuilderExtensions.cs#L18